src/utils/components/Consoles/components/CloudInitCredentials/CloudInitCredentialsContent.tsx

```diff
@@ -28,8 +30,12 @@ const CloudInitCredentialsContent: React.FC<CloudInitCredentialsContentProps> =
         for more information.
       </Trans>
       <div>
-        <strong>{t('User name: ')}</strong>
-        {user || CLOUD_INIT_MISSING_USERNAME}
+        {usernames.map((username) => (
```

Comment:
it's nice to show all the possible users, but if, for example, we will have 10 users, it will be a long list on separate lines... I'm not sure if we want to show them all or simply the first one. could you please consult with Yifat? 

Another thing, currently, the appearance is user: XXX password: XXX, in a single line from the last screenshot in the PR I can see its now separate in 2 lines. please check with her also this. - or just keep it consist that if its a single user it will be single line as its now

Reply:
Haven't contacted Yifat yet but also I think we don't have a design for this, and this is why we need to discuss the possible solutions.

Regarding another thing, username and password were already displayed the same way before I made any code changes, How to display them, that's another question for @yfrimanm. But I think we are fine because when both the username and password are present, there are no more any usernames, so no any problem about how to display it, username and password can be in a separate lines - better readability. Only when there is `passwd` object specified in the yaml, there can be more users and various credentials specified for them in the yaml (but not `password`).

Reply:
Found that I was mistaken and you were correct: originally for one user and password, both strings were displayed on one line. So I am fixing this, too. Thanks!